front/app/components/admin/ResourceList/SortableRow.tsx

```diff
@@ -45,12 +48,23 @@ class SortableRow extends React.Component<Props, State> {
     return connectDropTarget(
       connectDragSource(
         <div style={{ opacity }} className={className}>
-          <Row isLastItem={isLastItem}>
-            <DragHandle className="sortablerow-draghandle">
-              <Icon name="sort" />
-            </DragHandle>
-            {children}
-          </Row>
+          {noStyling && (
```

Comment:
Since we got rid of the drag-and-drop interface this modification is actually not necessary anymore, but I left it in because it can be useful in the future